### ISSUE TITLE
Can't login to WordPress if use H2O1.4.1.

### ISSUE THREAD

**Issue by AssemblyXiang** [h2o/h2o#425]
When I use H2O1.4.1 and login to WordPress,but can't login,and not show any error,just return to login page.

**Comment by kazuho** [comment]
Thank you for reporting the issue.

Have you tried using prior versions of H2O, or are you seeing the problem only on version 1.4.1? 

maybe related to #424

**Comment by AssemblyXiang** [comment]
I try version 1.3.1,but ploblem still exist.
this is my H2O config content:
```yaml
access-log: /tmp/acs.log
error-log: /tmp/err.log
pid-file: /tmp/h2o-1.4.1.pid
max-connections: 51200
file.send-gzip: ON
tcp-fastopen: 3
user: www
http2-reprioritize-blocking-assets: ON
file.index: [ 'index.php', 'index.html' ]

hosts:
  "www.asm-x.com:443":
    header.set: "strict-transport-security: max-age=39420000; includesubdomains; preload"
    listen:
      port: 443
      ssl:
        certificate-file: /home/wwwroot/ssl/asm-x.com.crt
        key-file: /home/wwwroot/ssl/asm-x.com.key
        dh-file: /home/wwwroot/ssl/dhparam4096.pem
        minimum-version: TLSv1
        cipher-preference: server
        cipher-suite: CHACHA20 EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH+aRSA+RC4 EECDH EDH+aRSA !aNULL !eNULL !LOW !3DES !MD5 !EXP !PSK !SRP !DSS !RC4
    paths:
      /:
        file.dir: /home/wwwroot/domain/asm-x.com
        file.custom-handler:
          extension: .php
          fastcgi.connect:
            port: /tmp/php-cgi-asm-x.com.sock
            type: unix
```

**Comment by kazuho** [comment]
Thank you for the details.  The issue has been fixed in #427.

**Comment by AssemblyXiang** [comment]
Oh,Thank you for quick fixed bug,and have a good day!
